libs/designer-ui/src/lib/combobox/index.tsx

```diff
@@ -94,6 +94,9 @@ export const Combobox = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [isLoading]);
 
+  // Sort newOptions array alphabetically based on the `displayName` property.
```

Comment:
Can we move this logic into libs\designer\src\lib\core\utils\parameters\helper.ts because, on the designer-ui side, we want to render the UI based on whatever props are passed in, and avoid manipulating the parameters as much as possible. This would require the sort to be done in 2 places: 
1. getParameterEditorProps (for initializing combobox) - when we set the options in editorOptions (line 398)
2. loadDynamicValuesForParameter (for initializing dynamic data in combobox) - before passing the editorOptions in updateNodeParaemters (2054)